src/Domain/LeanCode.DomainModels.EF/DateConverter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace LeanCode.DomainModels.EF
+{
+    public class DateConverter : ValueConverter<Date, DateTime>
+    {
+        public static readonly DateConverter Instance = new DateConverter { };
+        public DateConverter()
+            : base(d => d.ToDateTimeAtMidnight(), d => d.GetDate(), null)
```

Comment:
Split it into three lines, it will be more readable.

Also, you've missed a new line above.